src/Cell.cpp

```diff
@@ -1041,10 +1036,47 @@ void Cell::ringify(std::vector<Cell*>* subcells) {
                "the 2 halfspaces for each Surface.", _id, halfspace1,
                zcylinder1->getId(), halfspace2, zcylinder2->getId());
 
+  /* Loop over ZCylinders and create a new Cell clone for each ring */
+  std::vector<ZCylinder*>::iterator iter2;
+  std::vector<Cell*>::iterator iter3;
+
+  /* If this is an unbounded Cell - i.e., moderator in a fuel pin cell - then
+   * make the first ring span the volume outside the outermost cylinder */
+  if (radius1 > radius2) {
```

Comment:
Alright @samuelshaner @geogunow I tried to refactor this to achieve the equal volume invariance - let me know what you think. Note that this fix was quick - I simply merge what were previously the outermost two "rings" in the moderator. If you want something else, propose some code.